### PR TITLE
docs: add <meta> tags (Group 2)

### PR DESCRIPTION
**Pull request by marcpicaud**
Hey there!

## Summary
This PR adds `<meta>` tags to the pages listed as "Group 2" in #791 .

I kept the descriptions under 150 chars to avoid them being truncated, and I tried to keep them as descriptive, catchy and SEO friendly as possible. Feedback welcome if something can get improved !

### Location

- [Feature Guides](https://docs.flybywiresim.com/fbw-a32nx/feature-guides/) and all sub-pages within
- [API and Hardware](https://docs.flybywiresim.com/fbw-a32nx/a32nx-api/) and all the sub-pages within

### CODE REVIEW

File: docs/fbw-a32nx/feature-guides/cFMS.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+---
+title: Custom Custom Flight Management System
+description: Dive into the A32NX's custom FMS and stay informed with the latest known issues. Get tips for effective flight planning and troubleshooting.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/cFMS.md]:
```suggestion
description: Dive into the A32NX's custom FMS and stay informed with the latest known issues and troubleshooting tips.
```

Just to note there isn't much flight planning tips here and more of just information. Hence the suggestion.

File: docs/fbw-a32nx/feature-guides/flight-planning.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+---
+title: Flight Planning
+description: Discover how to load a flight plan into the FlyByWire A32NX across various methods, and seamlessly integrate it with your chosen ATC service.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/flight-planning.md]:
```suggestion
description: Discover how to load a flight plan into the FlyByWire A32NX across various methods, and integrate it with your chosen ATC service.
```

More notes: the MSFS linking isn't very seamless and has it's own issues just removing that would help.

File: docs/fbw-a32nx/feature-guides/flypados3/charts.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+---
+title: flyPadOS 3 EFB - Navigation & Charts
+description: Learn how to access and utilize charts on FlyPadOS3 with a Navigraph account or local files.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/flypados3/charts.md]:
```suggestion
description: Learn how to access and utilize charts on flyPadOS 3 with a Navigraph account or local files.
```

branding

File: docs/fbw-a32nx/feature-guides/flypados3/dashboard.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
+---
+title: flyPadOS 3 EFB - Dashboard
+description: A comprehensive guide on how to use the Dashboard widgets in FlyPadOS3 for the FlyByWire A32NX.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/flypados3/dashboard.md]:
```suggestion
description: A comprehensive guide on how to use the Dashboard widgets in flyPadOS 3 for the FlyByWire A32NX.
```

branding

File: docs/fbw-a32nx/feature-guides/flypados3/dispatch.md [hunk 5]
```diff
@@ -1,3 +1,8 @@
+---
+title: flyPadOS 3 EFB - Dispatch
+description: Information about the Dispatch menu provided by FlyPadOS3 in the FlyByWire A32NX.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/flypados3/dispatch.md]:
```suggestion
description: Information about the Dispatch menu provided by flyPadOS 3 in the FlyByWire A32NX.
```

File: docs/fbw-a32nx/feature-guides/flypados3/ground.md [hunk 6]
```diff
@@ -1,3 +1,8 @@
+---
+title: flyPadOS 3 EFB - Ground
+description: A comprehensive guide on how to use the ground services provided by FlyPadOS3 in the FlyByWire A32NX.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/flypados3/ground.md]:
```suggestion
description: A comprehensive guide on how to use the ground services provided by flyPadOS 3 in the FlyByWire A32NX.
```

File: docs/fbw-a32nx/feature-guides/flypados3/online-atc.md [hunk 7]
```diff
@@ -1,3 +1,8 @@
+---
+title: flyPadOS 3 EFB - Online ATC
+description: Discover how to use online ATC services such as VATSIM or IVAO with FlyPadOS3 for the FlyByWire A32NX.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/flypados3/online-atc.md]:
```suggestion
description: Discover how to use online ATC services such as VATSIM or IVAO with flyPadOS 3 for the FlyByWire A32NX.
```

File: docs/fbw-a32nx/feature-guides/flypados3/performance.md [hunk 8]
```diff
@@ -1,3 +1,8 @@
+---
+title: flyPadOS 3 EFB - Performance
+description: A comprehensive guide covering the descent and landing performance calculators provided by FlyPadOS3 in the FlyByWire A32NX.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/flypados3/performance.md]:
```suggestion
description: A comprehensive guide covering the descent and landing performance calculators provided by flyPadOS 3 in the FlyByWire A32NX.
```

File: docs/fbw-a32nx/feature-guides/flypados3/settings.md [hunk 9]
```diff
@@ -1,3 +1,8 @@
+---
+title: flyPadOS 3 EFB - Settings
+description: Learn how to configure various aspects of flyPadOS3 itself in the FlyByWire A32NX, from aircraft options to 3rd party settings.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/flypados3/settings.md:3]:
```suggestion
description: Learn how to configure various aspects of flyPadOS 3 itself in the FlyByWire A32NX, from aircraft options to 3rd party settings.
```

**Review by Valastiri — CHANGES_REQUESTED** [review]
some suggestions and branding of our flyPadOS 3 changes

**Comment by marcpicaud** [comment]
Hi there,

Appreciate the time you took to review the texts, thank you!
I've pushed the suggestions just now, sorry for the delay :)

Cheers

**Comment by Valastiri** [comment]
No worries and lgtm! Appreciate it.

**Review by Valastiri — APPROVED** [review]

**Pull request merged**